assignment/scripts/loops-practice.js

```diff
@@ -11,27 +11,57 @@ for (let i=0; i<4; i++) {
 }
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// I need to change the middle part. Change number for to number 6.
 console.log('count from 0 to 5');
 
+for (let i=0; i<6; i++) {  
```

Comment:
FWIW, you usually see spaces around these operators (other than `++`).

```suggestion
for (let i = 0; i < 6; i++) {  
```